q2_moshpit/citations.bib

```diff
@@ -129,3 +129,20 @@ @misc{NCBI
   url = {https://www.ncbi.nlm.nih.gov/},
   note = {Bethesda (MD): National Library of Medicine (US), National Center for Biotechnology Information;},
 }
+
+@article{huerta-cepas_eggnog_2019,
```

Comment:
I know it's because of the name, but let's not mix `-` and `_` characters - it's confusing and this is anyway just an ID so we may as well stick with `_`.